spec/renderer/integration/store/TimelineSpace/Contents/Mentions.spec.js

```diff
@@ -0,0 +1,152 @@
+import Mastodon from 'megalodon'
+import { createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import Mentions from '~/src/renderer/store/TimelineSpace/Contents/Mentions'
+
+jest.genMockFromModule('megalodon')
+jest.mock('megalodon')
+
+let state = () => {
+  return {
+    lazyLoading: false,
+    heading: true,
+    mentions: [],
+    unreadMentions: [],
+    filter: ''
+  }
+}
+
+const initStore =() => {
+  return {
+    namespaced: true,
+    state: state(),
+    actions: Mentions.actions,
+    mutations: Mentions.mutations
+  }
+}
+const timelineState = {
+  namespaced: true,
+  state: {
+    account: {
+      accessToken: 'token',
+      baseURL: 'http://localhost'
+    }
+  }
+}
+
+describe('Mentions', () => {
+  let store
+  let localVue
+
+  beforeEach(() => {
+    localVue = createLocalVue()
+    localVue.use(Vuex)
+    store = new Vuex.Store({
+      modules: {
+        Mentions: initStore(),
+        TimelineSpace: timelineState
+      }
+    })
+    Mastodon.mockClear()
+  })
+
+  describe('fetchMentions', () => {
+    it('should be updated', async () => {
+      const mockClient = {
+        get: () => {
+          return new Promise((resolve, reject) => {
+            resolve({
+              data: [
+                { id: 1, type: 'mention' },
+                { id: 2, type: 'favourite' },
+                { id: 3, type: 'reblog' },
+                { id: 4, type: 'follow' }
+              ]
+            })
+          })
+        }
+      }
+
+      Mastodon.mockImplementation(() => mockClient)
+      const mentions = await store.dispatch('Mentions/fetchMentions')
```

Comment:
__[ESLint]__ Error - 'mentions' is assigned a value but never used. (no-unused-vars) ([link](https://sider.review/gh/repos/124234495/pulls/849#issue-3855131))

<sub>posted by Sider</sub>